tools/dashboard-plugin-template/README.template.md

```diff
@@ -72,13 +72,36 @@ Building a new plugin is easy. Before you start, ensure that your `.env` and `.e
     _BAD IDEA_: overwriting existing plugin artifacts will immediately impact all dashboards that use the plugin, possibly
     breaking them if you did not have chance to fully test the plugin.
 
-5.  Add plugin to one or more workspaces: `{{packageManager}} dashboard-plugin add`
+5.  Add plugin to one or more workspaces: `{{packageManager}} run add-plugin -- <url>`
 
-    TBD
+    Once your plugin is uploaded to public hosting location, you can add it into your workspace. You can achieve this
+    using the same CLI tool that you have used to create this plugin project. For convenience, this project contains
+    the tool among the devDependencies together with convenience script to add plugin to either workspace specified
+    in your `.env` file (default) or another workspace that you specify on the command line.
 
-6.  Use plugin on a dashboard: `{{packageManager}} dashboard-plugin use`
+    Run the `{{packageManager}} run add-plugin -- "https://your.hosting/pluginDirOfYourChoice/dp_{{pluginIdentifier}}.js"` to
+    create a new dashboard plugin object in the workspace specified in the `.env` file. The created dashboard object
+    point to the URL of the built plugin.
 
-    TBD
+    After successful creation of the plugin object, the tool will print plugin object identifier. You will need this
+    identifier later to link dashboard(s) with the plugin.
+
+    Note: the CLI tool has options that allow you to add plugin to different backends and/or different workspaces. Check out
+    `{{packageManager}} gdc-plugins -- --help` to learn more about the tool's commands and options.
```

Comment:
Will this work with npm? I know `yarn dependency` will run it, not so sure about npm though.

Reply:
fixed.. i forgot to add `run` there same as it is in the other commands